packages/data-stores/src/site/reducer.ts

```diff
@@ -38,25 +43,40 @@ const newSiteError: Reducer< NewSiteErrorResponse | undefined, Action > = ( stat
 	return state;
 };
 
-const isFetchingSite: Reducer< boolean | undefined, Action > = ( state = false, action ) => {
+export const isFetchingSite: Reducer< boolean | undefined, Action > = ( state = false, action ) => {
 	switch ( action.type ) {
 		case 'FETCH_NEW_SITE':
 			return true;
 		case 'RECEIVE_NEW_SITE':
-			return false;
 		case 'RECEIVE_NEW_SITE_FAILED':
+		case 'RESET_SITE_STORE':
 			return false;
 	}
 	return state;
 };
 
+export const sites: Reducer< { [ key: string ]: SiteDetails | undefined }, Action > = (
```

Comment:
Should be

```suggestion
export const sites: Reducer< { [ key: number ]: SiteDetails | undefined }, Action > = (
```

now that we're keying by site ID :slightly_smiling_face:

Reply:
Oh, good catch! I've fixed that one up :)